### PR TITLE
Sorted session in SwapOptionsConfigurator to show mainnets first and testnets last.

### LINKED ISSUE #6543: Modify mainnets and testnets display order so that mainnets go before testnets in Swap options
Maybe already included with #6533

1. Wallet tab
2. Tap ETH mainnet
3. Tap …
4. Tap Swap
5. Tap Native Swap
6. Tap gear button

### PR DESCRIPTION
**Pull request by eviltofu**
Closes #6543 
Before | After
-|-
before|after

### CODE REVIEW

**Review by hboon — CHANGES_REQUESTED** [review]
Can you write it like this instead?

```
sessions = functional.sortSessionsWithMainnetsFirst(sessions: sessions)
invalidateSessionsWhenSupportedTokensChanged()
fetchSupportedTokensForSelectedServer()
resetToTokenForNonSupportedSwapPair()
```

Doesn't flow as well as your original version, but goes towards gradually reduce state-modifying code (mentioned in #6606)

**Comment by eviltofu** [comment]
Review @hboon

File: modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/Helpers/SwapOptionsConfigurator.swift [hunk 1]
```diff
@@ -284,3 +284,26 @@ public extension SwapOptionsConfigurator {
         return .just(nil)
     }
 }
+
+public extension SwapOptionsConfigurator {
```
Comment (hboon) [inline modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/Helpers/SwapOptionsConfigurator.swift:288]: Shouldn't be `public`

File: modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/Helpers/SwapOptionsConfigurator.swift [hunk 2]
```diff
@@ -284,3 +284,26 @@ public extension SwapOptionsConfigurator {
...
+    class functional {}
+}
+
+public extension SwapOptionsConfigurator.functional {
```
Comment (hboon) [inline modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/Helpers/SwapOptionsConfigurator.swift:292]: Shouldn't be `public`

**Review by hboon — APPROVED** [review]

File: modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/Helpers/SwapOptionsConfigurator.swift (hunk 2 above)
Comment (eviltofu) [reply modules/AlphaWalletFoundation/AlphaWalletFoundation/SwapToken/NativeSwap/Helpers/SwapOptionsConfigurator.swift]: Removed

**Pull request merged**